plugin/src/main/java/org/opensearch/sql/plugin/transport/TransportQueryAction.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.plugin.transport;
+
+import org.opensearch.action.ActionListener;
+import org.opensearch.action.ActionRequest;
+import org.opensearch.action.support.ActionFilters;
+import org.opensearch.action.support.HandledTransportAction;
+import org.opensearch.client.Client;
+import org.opensearch.common.inject.Inject;
+import org.opensearch.common.util.concurrent.ThreadContext;
+import org.opensearch.commons.ConfigConstants;
+import org.opensearch.commons.authuser.User;
+import org.opensearch.commons.sql.action.SQLActions;
+import org.opensearch.commons.sql.action.TransportQueryRequest;
+import org.opensearch.commons.sql.action.TransportQueryResponse;
+import org.opensearch.commons.sql.model.QueryType;
+import org.opensearch.tasks.Task;
+import org.opensearch.transport.TransportService;
+import org.opensearch.commons.utils.TransportHelpersKt;
+
+import java.io.IOException;
+
+/**
+ * Send SQL/PPL query transport action
+ */
+public class TransportQueryAction extends HandledTransportAction<ActionRequest, TransportQueryResponse> {
```

Comment:
should we 
1. create TransportSQLService and TransportPPLService
2. simply RestXXXAction

Reply:
good point, renames classed and updated